fdbserver/DDTeamCollection.actor.cpp

```diff
@@ -3328,15 +3328,19 @@ void DDTeamCollection::updateCpuPivots() {
 }
 
 void DDTeamCollection::updateTeamEligibility() {
+	int healthyCount = 0, lowDiskCount = 0, lowCpuCount = 0;
```

Comment:
There could be overlap between teams have low space and high cpu. For example, if we see 4 as low space, and 6 as high cpu, this could be a total of 10 teams that have either low space or high cpu if there is no overlap but could be as low as 6 teams if all the teams with low space also have high cpu. We could track this as `healthyCount`, `lowDiskUtilOnly`, `lowCpuOnly`, and `lowDiskLowCpu`, OR `healthyCount`, `lowDiskUtilTotal`, `lowCpuTotal`, and `lowDiskUtilLowCpuOverlap`.

Reply:
Also, I wonder if it would be more intuitive to track and trace the teams would be ineligible (i.e., the `highDiskUtil` and `highCpu`).

Reply:
I think it is more straightforward with `lowDisk...` and `lowCpu...` since it can directly indicate how many teams we have added to the eligible pool.